aspnetcore/fundamentals/servers/yarp/header-guidelines.md

```diff
@@ -45,7 +45,7 @@ This response header is used with HTTP/3 upgrades and only applies to the immedi
 ### Distributed tracing headers
 
 These headers include TraceParent, Request-Id, TraceState, Baggage, Correlation-Context.
-They are automatically removed based on `DistributedContextPropagator.Fields` so that the forwarding HttpClient can replace them with updated values.
+They are automatically removed based on `DistributedContextPropagator.Fields`, allowing the forwarding HttpClient to replace them with updated values.
```

Comment:
FYI ... The style manual calls for use of contractions. We usually only avoid them when giving weight to the word "not," for example, "Do ***not*** put your client secret in a public configuration file."

Also, if you use the [XREF Gen tool](https://dotnet.github.io/blazor-samples/), you can get the API cross-links quickly.

```suggestion
They're automatically removed based on <xref:System.Diagnostics.DistributedContextPropagator.Fields%2A?displayProperty=nameWithType>, allowing the forwarding <xref:System.Net.Http.HttpClient> to replace them with updated values.
```